website/docs/patterns/form-patterns/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,236 @@
+
+### Sections
+
+A **section** occupies the largest hierarchy within the form and is used as a descriptive term for organizing combinations of text, form elements (inputs, checkboxes, toggles, etc), and actions. Even in the simplest of circumstances a form can consist of multiple sections with differing content, each receiving a 32px vertical gap in between.
+
+![Example of multiple sections in a form](/assets/patterns/form-patterns/form-sections.png =450x*)
+
+### Fields
+
+A **field** is a broad term used to describe a form control and label pairing. When combined within a section there should be a 24px gap between each field.
+
+![Example of multiple fields in a section](/assets/patterns/form-patterns/section-fields.png =450x*)
+
+Fields can be further broken down into two types; text fields and data fields, each with it's own recommended usage and examples.
+
+#### Text fields
+
+<!-- TODO -->
+
+| Form control | Usage | Examples |
+|--------------|-------|----------|
+| [Text input](/components/form/text-input) | lorem ipsum | Name, email, password, string and numerical data |
+| [Textarea](/components/form/textrea) | Lorem ipsum | Messages, longer-form content |
+
+#### Data fields
+
+<!-- TODO -->
+
+| Form control | Usage | Examples |
+|--------------|-------|----------|
+| [Checkbox](/components/form/checkbox) | Lorem ipsum | Lorem ipsum |
+| [Radio](/components/form/radio) | Lorem ipsum | Lorem ipsum |
+| [Radio Card](/components/form/radio-card) | Lorem ipsum | Lorem ipsum |
+| [Select](/components/form/select) | Lorem ipsum | Lorem ipsum |
+| [Toggle](/components/form/toggle) | Lorem ipsum | Lorem ipsum |
+
+### Groups
+
+Fields may be **grouped** together either horizontally or vertically if they are collecting related information or information that is part of the same object. Within a group there should bea  16px gap either horizontally (creating multiple columns), or vertically between each field.
+
+![Example of fields grouped together](/assets/patterns/form-patterns/field-groups.png =450x*)
+
+!!! Dont
+
+Don't group (either horizontally or vertically) field types that are related but not of the same type, e.g., a text input and a toggle.
+
+![Grouping different field types](/assets/patterns/form-patterns/different-field-type.png)
+!!!
+
+!!! Do
+
+Instead, stack the fields vertically. 
+
+![Grouping different field types](/assets/patterns/form-patterns/different-field-type-do.png =500x*)
+!!!
+
+Common grouping of fields can include:
+
+- Credit card information; card number, expiration data, security code.
+- Multi-line address field
+- First and last name
+
+#### Alignment
+
+Fields organized in a horizontal group should be aligned to the baseline of each element to account for fields which have helper text.
+
+![Baseline alignment within a group](/assets/patterns/form-patterns/baseline-alignment.png =500x*)
+
+#### Multi-column layout
```

Comment:
I see an opportunity to display visual do/don't blocks in this section to help consumers better comprehend this content. It can be a follow-up task, but I wanted to note it anyway because I think it would be a beneficial addition.

Reply:
Yep, that's a good idea. I added visuals and do/dont blocks here.